cypress/integration/features/deals/partnered-deal/partner-dao.feature

```diff
@@ -1,37 +1,37 @@
 Feature: Create Partnered Deal Partner DAO stage
 
-  Background:
-    Given I navigate to the "Partnered Deal" "Partner DAO" stage
+  # Background:
+  #   Given I navigate to the "Partnered Deal" "Partner DAO" stage
 
-  Scenario: Partner DAO stage allows to add information about DAO and its representatives
-    Then I can see Partner DAO section with inputs for collecting its details
-    And I can see Partner DAO Representatives section
+  # Scenario: Partner DAO stage allows to add information about DAO and its representatives
+  #   Then I can see Partner DAO section with inputs for collecting its details
```

Comment:
@dkent600 
The files commented out: Tests were essentially the same.
Also for the first 2 stages, nothing fancy is happening, ie. nothing only for a particular wizard.

May be different for "Make an offer", but we don't have tests for this wizard in the first place.

--> Good (?) to remove tests, that don't add value

Reply:
yes good